Makefile

```diff
@@ -106,15 +107,20 @@ build-container:
 $(MANPAGES): %: %.md
 	@sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' $<  | $(GOMD2MAN) -in /dev/stdin -out $@
 
-docs: $(MANPAGES)
+.PHONY: image.docs
+image.docs:
+	@git clone https://github.com/containers/image.git image.docs 2>/dev/null || true
+	@(cd image.docs; git checkout ${IMAGEVERSION} -b ${IMAGEVERSION}; git filter-branch --prune-empty --subdirectory-filter docs -- --all 2>/dev/null;for i in *.md; do $(GOMD2MAN) -in $$i -out $${i//.md}; done) || true
```

Comment:
Could this clone only the `IMAGEVERSION` commit (`-depth 1`) instead of all of `master`? Or is that not possible with non-tag commit references?

Why the `filter-branch` instead of just using `docs/*.md` or something like that?

Reply:
Because that is what google found... :^)

Reply:
--depth=1 will not work,because of the IMAGEVERSION stuff.
I just aded the filter-branch to make it easier to debug.  But I could remove it if you don't like it.

Reply:
\*shrug\* I can live with all this, primarily I’d like @lsm5 to confirm that this is useful for packaging.

Reply:
WRT `filter-branch`, it seems unnecessarily expensive (O(repo history) compared to just using an explicit path at ~zero cost).

OTOH I _never_ run `make docs` so I am definitely not going to block this PR on performance optimizations.

Reply:
filter-branch removed